treeherder/config/settings.py

```diff
@@ -102,14 +102,9 @@ def server_supports_tls(url):
 
 AUTHENTICATION_BACKENDS = (
     'django.contrib.auth.backends.ModelBackend',
-    'django_browserid.auth.BrowserIDBackend',
+    'django.contrib.auth.backends.RemoteUserBackend',
```

Comment:
I'm pretty sure we need to be using login() from here:
https://docs.djangoproject.com/en/1.8/topics/auth/default/#how-to-log-a-user-in

And not the `RemoteUserBackend` function, which as far as I can tell is meant for when the `REMOTE_USER` HTTP header is set by a service in front of the app (eg an LDAP apache plugin in a corporate environment).